region/maridia/inner-pink/East Sand Hole.json

```diff
@@ -155,9 +146,29 @@
               "canSunkenTileWideWallClimb"
             ]}
           ]},
+          "canSpringBallJumpMidAir",
+          {"and": [
+            "h_canCrouchJumpDownGrab",
+            "canBombJumpWaterEscape",
+            "h_canIBJ"
+          ]}
+        ]}
+      ]
+    },
+    {
+      "link": [1, 3],
+      "name": "Suitless Bootless",
+      "requires": [
+        "canSuitlessMaridia",
+        "canPlayInSand",
+        {"or": [
           {"and": [
-            "HiJump",
-            "canSpringBallJumpMidAir"
+            "h_canMaxHeightSpringBallJump",
+            {"or": [
+              "canWalljump",
+              "can3HighMidAirMorph",
+              "h_canJumpIntoIBJ"
```

Comment:
```suggestion
              {"and": [
                "canBombJumpWaterEscape",
                "h_canJumpIntoIBJ"
              ]}
```

Reply:
This is done with spring ball, so it's a lot easier than what I think `canBombJumpWaterEscape` calls for. You can hold jump constantly and just place a well-timed bomb at the peak of any of the jumps